src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx

```diff
@@ -138,10 +138,174 @@ The Order and Usage Plan may contain defined terms that are denoted by capitaliz
+
+            <td>
+              $99
+            </td>
+          </tr>
+
+
+          <tr>
+            <td>
+              New Relic - Pro Full Platform Users<sup>3</sup>
+            </td>
+
+            <td>
+              1
+            </td>
+
+            <td>
+              Full Platform Users
+            </td>
+
+            <td>
+              $349
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              New Relic - Enterprise Full Platform Users<sup>3</sup>
+            </td>
+
+            <td>
+              1
+            </td>
+
+            <td>
+              Full Platform Users
+            </td>
+
+            <td>
+              $549
+            </td>
+          </tr>
 
 
+          <tr>
+            <td>
+              New Relic - Core Users
+            </td>
+
+            <td>
+              1
+            </td>
+
+            <td>
+              Core Users
+            </td>
+
+            <td>
+              $49
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              New Relic - Data<sup>3</sup>
+            </td>
+
+            <td>
+              1
+            </td>
+
+            <td>
+              GB Ingested
+            </td>
+
+            <td>
+              USD $0.30
+            </td>
+          </tr>
+
+          <tr>
+            <td>
+              New Relic - Data Plus<sup>3</sup>
+            </td>
+
+            <td>
+              1
+            </td>
+
+            <td>
+              GB Ingested
+            </td>
+
+            <td>
+              USD $0.50
+            </td>
+          </tr>
+
+
+          <tr>
+            <td>
+              Extended Retention for New Relic - Data and New Relic - Data Plus
+            </td>
+
+            <td>
+              1
+            </td>
+
+            <td>
+              Per 30 Days
+            </td>
+
+            <td>
+              USD $0.05
+            </td>
+          </tr>
+
+        </tbody>
+      </table>
+
+Notes:
+<sup>1</sup> Your Order and/or invoice will include both the Product name and applicable edition for your subscription.
+<sup>2</sup> List Prices are promotional and subject to change, and the prices above are for non-Pay As You Go plans and Orders with annual billing terms. List Prices for the New Relic Pro Full Platform Users, New Relic Enterprise Full Platform Users, and New Relic - Core Users shall be an additional 12% for non-Pay As You Go plans with monthly billing terms. List Prices for New Relic Pro Full Platform Users and New Relic Enterprise Full Platform Users shall be an additional 20% for Pay As You Go plans. All amounts are in USD.
+<sup>3</sup> You may only utilize one of the following user editions at any given time: New Relic - Standard Full Platform Users, New Relic - Pro Full Platform Users, or New Relic - Enterprise Full Platform Users. You may only utilize one data option at any given time: New Relic - Data or New Relic - Data Plus.
```

Comment:
@zuluecho9 what does this mean? Are you saying that companies can't have a mix of different types of users?

Also, I'd insert a carriage return between this footnote and #2 to keep them separate.

Reply:
that's just saying they can't have more than one edition at same time (they bill by users though)